packages/node_modules/@ciscospark/widget-recents/src/enhancers/setup.test.js

```diff
@@ -135,6 +150,30 @@ describe('widget-recents: enhancers: setup', () => {
         });
       });
 
+      describe('with extendedLoad disabled', () => {
```

Comment:
these will be removed soon, but still wanted to have test coverage until they aren't